assets/js/romo/base.js

```diff
@@ -724,7 +725,8 @@ RomoParentChildElems.prototype._push = function(items, id) {
   if (this.elems[id] === undefined) {
     this.elems[id] = []
   }
-  items.forEach($.proxy(function(item){ this.elems[id].push(item) }, this));
+  this.elems[id] = this.elems[id].concat(items);
+
```

Comment:
@jcredding I changed this to use `concat`, FYI.  I should have used this originally but didn't know about it at the time.